python/lsst/rapid/analysis/rubinTv.py

```diff
@@ -352,7 +343,7 @@ def callback(self, dataId):
             self.log.info('Upload complete')
 
         except Exception as e:
-            self.log.info(f"Skipped imExam on {dataId} because {e}")
+            self.log.warn(f"Skipped specExam on {dataId} because {e}")
```

Comment:
`self.log.warning` otherwise you will get a deprecation method (assuming you aren't still using lsst.log)

Reply:
Thanks, I've fixed all instances of this across the package now too.